nornir/plugins/tasks/networking/netmiko_save_config.py

```diff
@@ -0,0 +1,26 @@
+from __future__ import unicode_literals
+
+from nornir.core.task import Result, Task
+
+
+def netmiko_save_config(
+    task: Task, cmd: str = "", confirm: bool = False, confirm_response: str = ""
+) -> Result:
+    """
+    Execute Netmiko save_config method
+    Arguments:
+        cmd(str, optional): Command used to save the configuration.
+        confirm(bool, optional): Does device prompt for confirmation before executing save operation
```

Comment:
Can you add `confirm_response` to the docstring? This would be the string that gets sent in response to confirmation if required (for example a "Y").